src/pages/loginView.js

```diff
@@ -31,13 +31,16 @@ export default function LoginView() {
       validLogin = false;
     }
 
-    // If user is authenticated, we can redirect to home
     if (validLogin) {
-      // If successful, can go to /home
+      // If authentication is successful, can go to /home
       if (await signIn(processedEmail, processedPassword)) {
         history.push("/home");
       } else {
+        // If authentication is unsuccessful, notify user and reset inputs
         alert("Invalid login");
+
+        setEmail("");
```

Comment:
Now that I think about it, most websites don't reset the email field when it's an invalid login

Reply:
Good thinking!